Packs/CommonPlaybooks/Playbooks/playbook-IP_Enrichment_-_Internal_-_Generic_v2.yml

```diff
@@ -379,6 +417,8 @@ tasks:
           accessor: Hostname
           transformers:
           - operator: uniq
+      UseReputationCommand:
+        simple: "False"
```

Comment:
maybe this should be empty? so it will take the default of the original playbook?
 or point to the `inputs.UseReputationCommand`?

Reply:
Okay, I'll leave it empty